app/services/audio/audio.ts

```diff
@@ -203,14 +204,11 @@ export class AudioService extends StatefulService<IAudioSourcesState> implements
 
   setFader(sourceId: string, patch: Partial<IFader>) {
     const obsFader = this.sourceData[sourceId].fader;
-
-    if (patch.deflection) obsFader.deflection = patch.deflection;
-    if (patch.mul) obsFader.mul = patch.mul;
+    if (![null, undefined].includes(patch.deflection)) obsFader.deflection = patch.deflection;
```

Comment:
`if (patch.deflection != null)` is equivalent.  This is the one case where it's ok to use `==` instead of `===` because it checks for both `null` and `undefined` and nothing else, which usually you treat the same.